sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -36,6 +36,24 @@ static pi_result QueueRelease(pi_queue Queue, pi_queue LockedQueue);
 
 namespace {
 
+// The following supports an experimental feature to import host ptrs
+// into USM, improving data transfer performance.
+// Environment variable control over the feature.
+static const bool USMHostPtrImport = [] {
+  const char *USMHostPtrImportStr = std::getenv("SYCL_USM_HOSTPTR_IMPORT");
+  const pi_uint32 USMHostPtrImportValue =
+      USMHostPtrImportStr ? std::atoi(USMHostPtrImportStr) : 0;
+  return USMHostPtrImportValue;
+}();
+// Pointers to functions that import/release host memory into USM
+static ze_result_t (*zexDriverImportExternalPointer)(ze_driver_handle_t hDriver,
+                                                     void *, size_t);
+static ze_result_t (*zexDriverReleaseImportedPointer)(ze_driver_handle_t,
+                                                      void *);
```

Comment:
why are these here?

Reply:
OK, moved into a new class that handles all aspects on L0 import.